### ISSUE TITLE
'await++' in top level parsed incorrectly

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#789]
```js
let await = 1;
await++;
```
qljs shouldn't error in the above code, but it does.
